public/app/features/alerting/unified/components/silences/SilenceTableRow.tsx

```diff
@@ -56,13 +56,17 @@ const SilenceTableRow: FC<Props> = ({ silence, className, silencedAlerts, alertM
         </td>
         <td className={styles.actionsCell}>
           {status.state === 'expired' ? (
-            <ActionButton icon="sync">Recreate</ActionButton>
+            <a href={`/alerting/silence/${silence.id}/edit`}>
```

Comment:
I ended up using `<Link />` instead of `<a />` because `<Link />` works in tests as well